src/foam/u2/view/TableView.js

```diff
@@ -70,13 +70,13 @@ foam.CLASS({
       background: #eee;
     }
 
-    ^vertDots {
+    ^ .vertDots {
       font-size: 20px;
       font-weight: bold;
       padding-right: 12px;
     }
 
-    ^noselect {
+    ^ .noselect {
```

Comment:
Why the change?

CSS engines are typically faster at matching explicit class names like `^noselect`, rather than having to match on parent/child class name relationships like `^ .noselect`

Reply:
I made that change because I want to use that CSS class in both `UnstyledTableView` and the `OverlayActionListView` but `^noselect` will tie it to the table view.

Maybe this should be moved to our global styles.

Reply:
I see.

The downside of sharing CSS like this is that OverlayActionListView is now no longer fully componentized, it _has_ to be rendered inside a TableView in order for it to behave properly, and this dependency is not obvious in any way to a reader of either model.

Personally I would rather just repeat the CSS in OverlayActionListView, _or_ add some top level FOAM stylesheet that is _always_ included and contains some utility classes like .noselect

I don't think we have such a global stylesheet yet, but it might be worth investigating if we have some CSS classes that we want to reuse like this.

Reply:
Good point about dependencies between components. I just copied the CSS to the new view and reverted the relevant changes to CSS selectors for the TableView.